pointcloud/basic_octree_utils_test.go

```diff
@@ -171,6 +172,7 @@ func validateBasicOctree(t *testing.T, bOct *BasicOctree, center r3.Vector, side
 	validateMetadata(t, bOct)
 
 	var size int
+	maxVal := math.Inf(-1)
```

Comment:
this might be a lil spicy.
can't use `var maxVal float64` as that is actually 0 and 0 > (negative value) - not good for testing on negative data values.